### DISCUSSION TITLE
Support GitHub Emoji :octocat: also on GitHub Mobile's Stars Lists

### DISCUSSION THREAD

**Discussion by ymw0407** [community/community#77829] [Mobile]
### Select Topic Area

General

### Body

![image](https://github.com/community/community/assets/77202633/29296461-66a0-423b-82c8-730655c32940)
The GitHub web interface supports 8 GitHub-owned emojis. It also supports some parts of the GitHub mobile app, but it is not yet supported by GitHub Star's List and so on! It would be great if you could support GitHub Emoji globally.
![image](https://github.com/community/community/assets/77202633/71d07197-7040-4c5f-88a8-5a69ce5da845)
![image](https://github.com/community/community/assets/77202633/dabefcd9-eecb-4ce2-bd17-1c5f0e8987c7)

**Comment by dakshithadissanayaka** [comment]
As of my last knowledge update in January 2022, GitHub does support a variety of emojis throughout its platform, including the web interface and the mobile app. These emojis are commonly used for reactions, comments, issues, pull requests, and more. However, GitHub may periodically update its features, and new developments might occur after my last update.

If there are specific features or areas of the GitHub platform where you are experiencing issues with emoji support, I recommend checking GitHub's official documentation, release notes, or community forums for the latest information and updates.

Additionally, if you have a feature request or feedback for GitHub, you may consider reaching out to GitHub Support or providing feedback through the appropriate channels. GitHub actively considers user feedback for platform improvements, and your input could contribute to future updates.

Keep in mind that information might have changed since my last update, so it's always a good idea to check the latest GitHub documentation or contact GitHub support for the most accurate and up-to-date information.

**Comment by iskae** [comment]
Hi @ymw0407 👋 

Thank you for reaching out ❤️ You are absolutely right, we are missing some GitHub owned emojis on mobile 😅 

Let me create an internal tracking issue for this 👍 Please stay tuned for our future releases 🙇 

Cheers!
